src/tpm2-tss-engine-rand.c

```diff
@@ -40,6 +40,46 @@
 #include "tpm2-tss-engine.h"
 #include "tpm2-tss-engine-common.h"
 
+/** rand seed
+ * @retval 1 on success
+ * @retval 0 on failure
+ */
+static int
+rand_seed(const void *seed, int seed_len)
+{
+    ESYS_CONTEXT *esys_ctx = NULL;
+    TSS2_RC r;
+
+    r = esys_ctx_init(&esys_ctx);
+    ERRchktss(rand_seed, r, goto end);
+
+    TPM2B_SENSITIVE_DATA stir;
+    size_t offset = 0;
+    char *cur_data = (char*)seed;
+
+    static const size_t tpm_random_stir_max_size = 128; 
```

Comment:
Unfortunately there is a trailing whitespace here.

Reply:
i fix it now.

Reply:
Sorry, but I need this to be one commit.
So you can just squash those two commits together and force-push them.